shotover-proxy/tests/helpers/cassandra.rs

```diff
@@ -498,7 +508,23 @@ impl CassandraConnection {
 
                 Self::process_cdrs_response(response)
             }
-            Self::Scylla { .. } => todo!(),
+            Self::Scylla { session, .. } => {
+                let statement = prepared_query.as_scylla();
+                let response = session.execute(statement, (value,)).await.unwrap();
+
+                if let Ok(rows) = response.rows() {
```

Comment:
We are silently ignoring the error here, while the other implementations panic on error here.

Reply:
https://docs.rs/scylla/latest/scylla/transport/query_result/struct.RowsExpectedError.html

We are ignoring this error, it only occurs when the rows are empty.

Reply:
Oh good point.
We can make the code less suspicious looking by directly accessing the rows field which is an Option.
![image](https://user-images.githubusercontent.com/5120858/200463532-23ec44eb-cf6b-4948-9b60-ef2a68b2139c.png)